docker-compose.yml

```diff
@@ -9,6 +9,9 @@
                                               # will cause persistence issues!!
       "./observatory-extension:/observatory-extension"
     ],
+    "ports": [
```

Comment:
Was this needed locally?  We don't want to hardcode this port as it messes with CI.